### ISSUE TITLE
--info=progress2: unknown option

### ISSUE THREAD

**Issue by snowe2010** [xxh/xxh#63]
**Host Version: macOS Mojave**
**Remote version: Ubuntu 16.04.6 LTS (GNU/Linux 4.4.0-1077-aws x86_64)**
**xxh version: xxh/0.6.2**
**xxh-plugins installed:**
```
 5685  pip3 install --upgrade xxh-xxh
 5687  xxhp i xxh-shell-zsh
 5689  xxhp i xxh-plugin-zsh-ohmyzsh
 5690  source xxh.zsh myhost +if +q
 5691  xxh -i ~/.ssh/... tyler.thrailkill@111.11.11.11
 5692  vim ~/.xxh/.xxhc
```

```
➜ cat ~/.xxh/.xxhc
hosts:
   ".*":                     # Regex for all hosts
     +s: xxh-shell-zsh
```

**Steps to Reproduce**
1. Ran `xxh -i ~/.ssh/jumpcloud tyler.thrailkill@111.11.11.11`
2. The output log:

```
➜ xxh -i ~/.ssh/jumpcloud tyler.thrailkill@111.11.11.11
111.11.11.11:/home/tyler.thrailkill/.xxh/xxh/shells/xxh-shell-zsh not found. Install xxh-shell-zsh? [Y/n] y
Install xxh-shell-zsh to 111.11.11.11:/home/tyler.thrailkill/.xxh
Upload using rsync
rsync: --info=progress2: unknown option
rsync error: syntax or usage error (code 1) at /BuildRoot/Library/Caches/com.apple.xbs/Sources/rsync/rsync-52.200.1/rsync/main.c(1337) [client=2.6.9]
rsync: --info=progress2: unknown option
rsync error: syntax or usage error (code 1) at /BuildRoot/Library/Caches/com.apple.xbs/Sources/rsync/rsync-52.200.1/rsync/main.c(1337) [client=2.6.9]
rsync: --info=progress2: unknown option
rsync error: syntax or usage error (code 1) at /BuildRoot/Library/Caches/com.apple.xbs/Sources/rsync/rsync-52.200.1/rsync/main.c(1337) [client=2.6.9]
First run xxh-shell-zsh on 111.11.11.11
bash: /home/tyler.thrailkill/.xxh/xxh/shells/xxh-shell-zsh/build/entrypoint.sh: No such file or directory
```

**For community:**
⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

**Comment by anki-code** [comment]
Hi @snowe2010 ! Thank you for report this!
Fixed in 0.6.3 please update xxh:
```
pip install --upgrade xxh-xxh
```

**Comment by snowe2010** [comment]
Fixed! Thanks!
